packages/css-library/src/tokens/color.scss

```diff
@@ -0,0 +1,67 @@
+// Our colors
+$tokens-color-basic: (
+  "base": "#212121",
+  "white": "#ffffff",
+  "black": "#000000",
+  "orange": "#eb7f29",
+  "link-default": "#004795",
+  "warning-message": "#fac922",
+  "gibill-accent": "#fff1d2",
+  "primary": "#0071bb",
+  "primary-darker": "#003e73",
+  "primary-darkest": "#112e51",
+  "primary-alt": "#02bfe7",
+  "primary-alt-dark": "#00a6d2",
+  "primary-alt-darkest": "#046b99",
+  "primary-alt-light": "#9bdaf1",
+  "primary-alt-lightest": "#e1f38",
+  "secondary": "#e31c3d",
+  "secondary-dark": "#cd2026",
+  "secondary-darkest": "#981b1e",
+  "secondary-light": "#e59393",
+  "secondary-lightest": "#f9dede",
+);
+
+$tokens-color-grayscale: (
+  "gray": "#5b616b",
+  "gray-dark": "#323a45",
+  "gray-medium": "#757575",
+  "gray-light": "#aeb0b5",
+  "gray-light-alt": "#eeeeee",
+  "gray-lighter": "#d6d7d9",
+  "gray-lightest": "#f1f1f1",
+  "gray-warm-dark": "#494440",
+  "gray-warm-light": "#e4e2e0",
+  "gray-cool-light": "#dce4ef",
+);
+
+$tokens-color-tertiary: (
+  "gold": "#fdb81e",
+  "gold-darker": "#988530",
+  "gold-light": "#f9c642",
+  "gold-lighter": "#fad980",
+  "gold-lightest": "#fff1d2",
+  "green": "#2e8540",
+  "green-darker": "#195c27",
+  "green-light": "#4aa564",
+  "green-lighter": "#94bfa2",
+  "green-lightest": "#e7f4e4",
+  "cool-blue": "#205493",
+  "cool-blue-light": "#4773aa",
+  "cool-blue-lighter": "#8ba6ca",
+  "cool-blue-lightest": "#dce4ef",
+);
+
+// TODO: Consider removing these since they aren't used
```

Comment:
I agree they are not in use but, they should be and those are the right colors for those hub sections. For example, the icon in use in the disability section should use the hub-disability color token.